taskcluster/focus_android_taskgraph/target_tasks.py

```diff
@@ -31,38 +31,11 @@ def filter(task, parameters):
     return [l for l, t in full_task_graph.tasks.items() if filter(t, parameters)]
 
 
-@_target_task("nightly")
+@_target_task('nightly')
 def target_tasks_nightly(full_task_graph, parameters, graph_config):
     """Select the set of tasks required for a nightly build."""
-    return [l for l, t in full_task_graph.tasks.items() if filter(t, parameters)]
-
-
-def _filter_fennec(fennec_type, task, parameters):
-    return task.attributes.get("build-type", "") == f"fennec-{fennec_type}"
-
-
-@_target_task("fennec-production")
-def target_tasks_fennec_nightly(full_task_graph, parameters, graph_config):
-    """Select the set of tasks required for a production build signed with the fennec key."""
-
-    return [l for l, t in full_task_graph.tasks.items() if _filter_fennec("production", t, parameters)]
-
-
-@_target_task("bump_android_components")
-def target_tasks_bump_android_components(full_task_graph, parameters, graph_config):
-    """Select the set of tasks required to update android components."""
-
-    def filter(task, parameters):
-        return task.attributes.get("bump-type", "") == "android-components"
-
-    return [l for l, t in full_task_graph.tasks.items() if filter(t, parameters)]
 
+    def filter(task):
+        return 'nightly' in task.label.split('-')
```

Comment:
I think it would be better if you did what Fenix does, and look at attributes for this (https://github.com/mozilla-mobile/fenix/blob/main/taskcluster/fenix_taskgraph/target_tasks.py#L30). This is a bit more brittle and implicit.

Reply:
The problem with what Fenix did is that `attributes: nightly: true` is only explicitly used in the builds kind and its not at all clear how signing and push-apk nightly tasks are run by the cron task. In my mind, this approach is more obvious. Nightly tasks are all clearly labeled, so are release and beta. If you were to insist that we should go by attribute instead of label, then I'd rename it to something less obscure like `attributes: nightly-task: true` but then I'd do that for everything, so every single kind would have an extra attribute (for release-task, for beta-task, etc).

Reply:
I added `nightly-task` attribute per our discussion in the relounge channel and tested it on the staging repo.